src/Service/SF1500/PersonFetchService.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace App\Service\SF1500;
+
+use App\Entity\SF1500\PersonRegistrering;
+use App\Entity\SF1500\PersonRegistreringEgenskab;
+use App\Service\SF1500Service;
+use Doctrine\ORM\EntityManagerInterface;
+use ItkDev\Serviceplatformen\SF1500\Person\ServiceType\_List;
+use ItkDev\Serviceplatformen\SF1500\Person\ServiceType\Soeg;
+use ItkDev\Serviceplatformen\SF1500\Person\StructType\EgenskabType;
+use ItkDev\Serviceplatformen\SF1500\Person\StructType\FiltreretOejebliksbilledeType;
+use ItkDev\Serviceplatformen\SF1500\Person\StructType\ListInputType;
+use ItkDev\Serviceplatformen\SF1500\Person\StructType\RegistreringType;
+use ItkDev\Serviceplatformen\SF1500\Person\StructType\SoegInputType;
+use Psr\Log\LoggerAwareTrait;
+
+class PersonFetchService implements FetchServiceInterface
+{
+    use LoggerAwareTrait;
+
+    public function __construct(private readonly EntityManagerInterface $entityManager, private readonly SF1500Service $sf1500Service)
+    {
+    }
+
+    public function fetch(int $pageSize, int $max): void
+    {
+        $total = 0;
+
+        while (true) {
+            $this->logger->debug(sprintf('Fetching person data, offset: %d , max: %d', $total, $max));
+            $this->logger->debug(sprintf('Memory used: %d ', memory_get_usage() / 1024 / 1024));
```

Comment:
Consider using something like https://www.codexworld.com/how-to/convert-file-size-to-human-readable-format-php/ to format the memory usage.

Reply:
Added a `convertFilesize` method.